apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -863,68 +880,89 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
+                                                AppConstants.getPaths()
+                                                    .get("ATTRIBUTE_MAPPINGS")
+                                                    .replace(":type", ClaimManagementConstants.OIDC)
+                                            );
+                                        } }
+                                    > Attributes.
+                                    </Link>
+                                </Trans>
+                            </Hint>
+                        </Grid.Column>
+                    </Grid.Row>
+                    <Grid.Row className="mt-5">
+                        <Grid.Column computer={ 16 } tablet={ 16 } largeScreen={ 12 } widescreen={ 12 }>
+                            <Grid.Row>
+                                <Form>
+                                    <Grid.Column>
+                                        <Form.Field>
+                                            <div className="display-flex">
+                                                <CopyInputField
+                                                    className="copy-input spaced"
+                                                    value={ selectedScopes.join(" ") }
+                                                    data-testid={ `${ testId }-client-id-readonly-input` }
+                                                />
+                                            </div>
+                                        </Form.Field>
+                                    </Grid.Column>
+                                </Form>
+                            </Grid.Row>
                         </Grid.Column>
                     </Grid.Row>
+                    <Hint>
+                        <Trans
+                            i18nKey={
+                                "console:develop.features.applications.edit.sections.attributes." +
+                                                "selection.selectedScopesComponentHint"
+                            }
+                        >
+                            Request these scopes from your application to retrieve the user attributes 
+                            you selected.
+                        </Trans>
+                        {
+                            applicationConfig.attributeSettings.advancedAttributeSettings
+                                .showHowToUseScopesDocLink
+                                ? ( <Trans
+                                    i18nKey={
+                                        "console:develop.features.applications.edit.sections.attributes." +
+                                                "selection.howToUseScopesHint"
+                                    }
+                                >
+                                    <DocumentationLink
+                                        link={ 
+                                            getLink("develop.connections.editApplication.attributeManagement" +
```

Comment:
Remove this variable and it will not display the link

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/1eea89c4b4ff909d642ad66bb648e32a18e60490